ui/src/api/utils.js

```diff
@@ -63,6 +63,33 @@ export const schemaToPidType = schema => {
   }
 };
 
+export const sequenceRelationPayload = (
+  referer,
+  selectedRelated,
```

Comment:
Do these terms `referrer`, and `seletedRelated`  pro exist? (for example from the generic component you have implemented) if not I would keep the same `previous`, `next`.

nit: referrer has a typo, needs double rr